troposphere/static/js/components/projects/detail/resources/InstanceActionButtons.jsx

```diff
@@ -32,6 +32,10 @@ export default React.createClass({
         modals.InstanceModals.resume(this.props.instance);
     },
 
+    onUnshelve: function() {
+        modals.InstanceModals.unshelve(this.props.instance);
```

Comment:
so we have this modal before. Not sure how we could use the modal before this patch. But generally, this is very handy 👍